go/vt/mysqlctl/grpcmysqlctlclient/client.go

```diff
@@ -84,6 +83,18 @@ func (c *client) RunMysqlUpgrade(ctx context.Context) error {
 	})
 }
 
+// ApplyBinlogFile is part of the MysqlctlClient interface.
+func (c *client) ApplyBinlogFile(ctx context.Context, binlogFileName, binlogRestorePosition string) error {
+	req := &mysqlctlpb.ApplyBinlogFileRequest{
+		BinlogFileName:        binlogFileName,
+		BinlogRestorePosition: binlogRestorePosition,
+	}
+	return c.withRetry(ctx, func() error {
+		_, err := c.c.ApplyBinlogFile(ctx, req)
```

Comment:
Not a fan of `c.c.` but unrelated to this PR

Reply:
Copied this basically from the other things we have to be consistent.